akka-docs/src/test/scala/docs/stream/operators/source/FutureSource.scala

```diff
@@ -10,7 +10,11 @@ import akka.NotUsed
 import akka.stream.scaladsl.Source
 
 import scala.concurrent.Future
+// #sourceFutureSource
```

Comment:
Is that for a new code snippet in the docs?

Reply:
No it's a sample that uses more than two such tags to make a single snippet